### DISCUSSION TITLE
Not so much a bug, but as someone new to Godot, just a thought that might help the engine

### DISCUSSION THREAD

**Discussion by Soliaonline** [godotengine/godot-proposals#6582] [GUI]
### Godot version

4.0

### System information

Window 10

### Issue description

When making a theme, it was very hard to find out that the most important button on the entire screen is the "Manage Items..." button.  After you manage items they all then appear in the inspector.  Maybe consider added manage items under RefCounted.  Many things in Godot have been extremely intuitive and this just hasn't been one of them.  Thank you for your time and for making such a great engine.

### Steps to reproduce

Go to page.

### Minimal reproduction project

Create a theme.

**Comment by YuriSizov** [comment]
"Manage Items..." is not the most important button though. It's for power-users who want to do some bulk changes to the theme. Your regular use would be to pick items that you need from the preview and edit the options that you want. There is rarely a need to override every theme property of every control, unless you are making a complete theme outside of a specific project.

See also https://docs.godotengine.org/en/stable/tutorials/ui/gui_using_theme_editor.html
